telegram/src/main/kotlin/com/github/kotlintelegrambot/network/serialization/adapter/InlineKeyboardButtonAdapter.kt

```diff
@@ -39,6 +40,7 @@ internal class InlineKeyboardButtonAdapter : JsonSerializer<InlineKeyboardButton
         is SwitchInlineQueryCurrentChat -> context.serialize(src, SwitchInlineQueryCurrentChat::class.java)
         is CallbackGameButtonType -> context.serialize(src, CallbackGameButtonType::class.java)
         is Pay -> context.serialize(src, Pay::class.java)
+        is WebApp -> context.serialize(src, WebApp::class.java)
```

Comment:
Is it worth adding a test case in `InlineKeyboardButtonAdapterTest` for this?

Reply:
- Added `InlineKeyboardButtonAdapterTest`
- Fixed `KeyboardButton`
- Added deserialization of WebApp

Please review and check if I messed something.
Regards